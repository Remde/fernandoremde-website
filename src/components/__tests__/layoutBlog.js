import React from "react"
import renderer from "react-test-renderer"

import LayoutBlog from "../layoutBlog.js"

describe("LayoutBlog", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LayoutBlog />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
