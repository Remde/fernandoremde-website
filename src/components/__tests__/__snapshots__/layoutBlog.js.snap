// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutBlog renders correctly 1`] = `
<div>
  <div
    className="navbar"
  >
    <div
      className="homeLink"
    >
      <a
        href="/"
      >
        home
      </a>
    </div>
    <div
      className="projectsLink"
    >
      <a
        href="#projects"
      >
        projects
      </a>
    </div>
    <div
      className="blogLink"
    >
      <a
        href="/blog"
      >
        blog
      </a>
    </div>
    <div
      className="aboutLink"
    >
      <a
        href="/about"
      >
        about
      </a>
    </div>
  </div>
  <div
    className="blogPosts"
    style={
      Object {
        "padding": "2.175rem 1.0875rem",
      }
    }
  >
    <header>
      <h3
        style={
          Object {
            "fontSize": 50,
            "marginTop": 40,
            "textAlign": "left",
          }
        }
      />
    </header>
    <main />
  </div>
  <div
    className="about"
    id="social-section"
  >
    <a
      href="https://www.github.com/Remde"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="github"
      />
    </a>
    <a
      href="mailto:ffremde@gmail.com"
    >
      <div
        className="mail"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/fernando-remde-291a641a6/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="linkedin"
      />
    </a>
  </div>
  <div
    className="footer"
  >
    <div
      className="text"
    >
      A website made by Fernando Ferreira Remde Â© | ffremde@gmail.com | Porto Alegre, Brazil
    </div>
  </div>
</div>
`;
